frontend/src/custom.d.ts

```diff
@@ -21,3 +23,9 @@ declare module '*.lottie' {
     const content: any
     export default content
 }
+
+declare module 'react' {
+    interface HTMLAttributes<T> extends AriaAttributes, DOMAttributes<T> {
+        inert?: ''
+    }
+}
```

Comment:
this change breaks CI because of how we import `.lottie` files.  [Example](https://github.com/PostHog/posthog/actions/runs/10462050717/job/28971579496?pr=24420)

What specifically is this `inert` attribute doing?  I pulled your branch and removed the modifications to this file and everything seemed to work fine.  so unless we need `inert` for functionality, my recommendation would be to remove these changes and just ship the changes to `SurveyTemplates.*`.

Reply:
Hi @dmarticus - sure, go ahead. The `inert` makes the child elements of each template deactivated so that they can't be clicked on or tabbed through, similar to putting `disabled` on each individual input, textarea etc.

Without it, the user will be able to tab through these again (for example, the 0-10 numbers on the likert template) but only when their click is active somewhere other than the body - e.g. the top nav (when they've come to the templates page from the 'New survey' button rather than the 'Create survey' button) or the side nav. As such, it's a bit of an edge case.

If they used the 'Create survey' button, or their click is active in the body of the page, then tabbing will simply tab through each of the templates with the focus as expected.

Reply:
Updated! @dmarticus